src/Mod/Path/PathScripts/PathDressupLeadInOut.py

```diff
@@ -33,46 +33,127 @@
 import math
 import copy
 
-from PySide import QtCore, QtGui
-
 __doc__ = """LeadInOut Dressup MASHIN-CRC USE ROLL-ON ROLL-OFF to profile"""
 
-if FreeCAD.GuiUp:
-    import FreeCADGui
+from PySide.QtCore import QT_TRANSLATE_NOOP
+
+from PathPythonGui.simple_edit_panel import SimpleEditPanel
 
-# Qt translation handling
-def translate(text, context="Path_DressupLeadInOut", disambig=None):
-    return QtCore.QCoreApplication.translate(context, text, disambig)
+translate = FreeCAD.Qt.translate
 
-PathLog.setLevel(PathLog.Level.INFO, PathLog.thisModule())
+if False:
+    PathLog.setLevel(PathLog.Level.DEBUG, PathLog.thisModule())
+    PathLog.trackModule(PathLog.thisModule())
+else:
+    PathLog.setLevel(PathLog.Level.INFO, PathLog.thisModule())
 
-movecommands = ['G1', 'G01', 'G2', 'G02', 'G3', 'G03']
-rapidcommands = ['G0', 'G00']
-arccommands = ['G2', 'G3', 'G02', 'G03']
+
+movecommands = ["G1", "G01", "G2", "G02", "G3", "G03"]
+rapidcommands = ["G0", "G00"]
+arccommands = ["G2", "G3", "G02", "G03"]
 currLocation = {}
 
 
 class ObjectDressup:
-
     def __init__(self, obj):
+        lead_styles = [
+            QT_TRANSLATE_NOOP("Path_DressupLeadInOut", "Arc"),
```

Comment:
I see that we're asking translators to provide translations for these strings, but I don't see where those translations are ever used. Can you point me to the place?

Reply:
They are used in the comboboxes in the gui. Labels for those are expected to be translatable.